src/index.ts

```diff
@@ -608,7 +608,7 @@ export class Validation implements FormValidation {
     if (!this.fieldsToValidate.includes(field))
       throw new Error(`Field "${field.name}" is not being validated`);
 
-    return !!!this.validateField(field, silently);
+    return !!this.validateField(field, silently);
```

Comment:
Just want to confirm if this is fine. I'm guessing since this was inside the same commits as some tests, it might've get carried over, but since this was later changed to `!this.validateField(field, silently);` I assume that change will stay still be there.

Reply:
yeah just remove the change to added in a different PR